core/client/hdfs/src/main/java/alluxio/hadoop/HadoopConfigurationUtils.java

```diff
@@ -30,35 +24,25 @@
  */
 @ThreadSafe
 public final class HadoopConfigurationUtils {
-  private static final Logger LOG = LoggerFactory.getLogger(HadoopConfigurationUtils.class);
-
   private HadoopConfigurationUtils() {} // Prevent instantiation.
 
   /**
-   * Merges Hadoop {@link org.apache.hadoop.conf.Configuration} with Alluxio properties.
+   * Extracts relevant configuration from Hadoop {@link org.apache.hadoop.conf.Configuration}.
    *
-   * @param hadoopConf the {@link org.apache.hadoop.conf.Configuration} to merge
-   * @param alluxioProps the Alluxio properties to merge
-   * @return a configuration with properties all merged
+   * @param hadoopConf the {@link org.apache.hadoop.conf.Configuration} to extract
+   * @return all relevant properties in a Map instance
    */
-  public static InstancedConfiguration mergeHadoopConfiguration(
```

Comment:
Presto was using this method 
https://github.com/prestodb/presto/blob/472538a02722c5409649533f9a602b5b76d9dd2d/presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCachingFileSystem.java#L98

Reply:
If we want to make these changes, we will need to fix this method
https://github.com/prestodb/presto/blob/472538a02722c5409649533f9a602b5b76d9dd2d/presto-cache/src/main/java/com/facebook/presto/cache/alluxio/AlluxioCachingFileSystem.java#L87

Reply:
ok, i will keep this method